frameworks/cassandra/tests/test_udf.py

```diff
@@ -0,0 +1,52 @@
+from typing import Any, Dict, Iterator, List
+import logging
+import pytest
+import sdk_jobs
+import sdk_install
+from tests import config
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module", autouse=True)
+def configure_package(configure_security: None) -> Iterator[None]:
+    test_jobs: List[Dict[str, Any]] = []
+    try:
+        test_jobs = config.get_udf_jobs()
+        # destroy/reinstall any prior leftover jobs, so that they don't touch the newly installed service:
+        for job in test_jobs:
+            sdk_jobs.install_job(job)
+
+        service_options = {
+            "service": {
+                "name": config.SERVICE_NAME,
+                "security": {"authentication": {"enabled": True}, "authorization": {"enabled": True}},
+            },
+            "cassandra": {
+                "enable_user_defined_functions": True,
+                "enable_scripted_user_defined_functions": True,
+            }
+        }
+
+        sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
```

Comment:
Can we reuse an existing Cassandra cluster from a different test? We're only testing UDFs (a simple one) - it seems a little cumbersome to un+install a fresh C* cluster just to test this feature. From TC log, it looks like this test alone adds an additional 3.5 minutes to the already very long CI job.

Reply:
added UDF test to test_sanity.py